topics/bloomfilters.md

```diff
@@ -0,0 +1,228 @@
+
+## Bloom properties
+
+* Capacity - The number of unique items that would need to be added before a scale out occurs or (non scaling) before it rejects addition of unique items. 
+
+* False Positive Rate (Error rate) - The rate that controls the probability of bloom check/set operations being false positives. Example: An item addition returning 0 (or an item check returning 1) indicating that the item was already added even though it was not.
+
+* Expansion - This is a property of scalable bloom filters which controls the growth in overall capacity when a bloom filter scales out by determining the capacity of the new sub filter which gets created. This new capacity is equal to the previous filters capacity * expansion rate 
+
+### Advanced Properties
+
+The following two properties can be specified in the `BF.INSERT` command:
+
+* Seed -  This is the key with which hash functions are created for the bloom filter. In case of scalable bloom filters, the same seed is used across all sub filters. This property is only useful if you have a specific 32 byte seed that you want your bloom filter to use. By default every bloom filter will use a random seed.
+
+* Tightening Ratio - This is a property of scalable bloom filters which controls the overall correctness of the bloom filter as it scales out by keeping the actual false positive rate closer to the user requested false positive rate when the bloom filter was created. This is done by using the tightening ratio to set a stricter false positive on the new sub filter which gets created during each scale out. We do not recommend fine tuning this unless there is a specific use case for lower memory usage with higher false positive or vice versa.
+
+### Default bloom properties
+
+<table width="100%" border="1" style="border-collapse: collapse; border: 1px solid black" cellpadding="8">
+<tr>
+<th width="30%">Property</th>
+<th width="30%">Default Value</th>
+<th width="40%">Engine Config (Global)</th>
+</tr>
+<tr>
+<td>Capacity</td>
+<td>100</td>
+<td>BF.BLOOM-CAPACITY</td>
+</tr>
+<tr>
+<td>False Positive Rate</td>
+<td>0.01</td>
+<td>BF.BLOOM-FP-RATE</td>
+</tr>
+<tr>
+<td>Scaling / Non Scaling</td>
+<td>Scaling</td>
+<td>BF.BLOOM-EXPANSION</td>
+</tr>
+<tr>
+<td>Expansion Rate</td>
+<td>2</td>
+<td>BF.BLOOM-EXPANSION</td>
+</tr>
+<tr>
+<td>Tightening Ratio</td>
+<td>0.5</td>
+<td>BF.BLOOM-TIGHTENING-RATIO</td>
+</tr>
+<tr>
+<td>Seed</td>
+<td>Random Seed</td>
+<td>BF.BLOOM-USE-RANDOM-SEED</td>
+</tr>
+</table>
+
+
+As bloom filters have a default expansion of 2 this means all default bloom objects will be scaling. These options are used when not specified explicitly in the commands used to create a new bloom object. For example doing a BF.ADD for a new filter will create a filter with the exact above qualities. These default properties can be configured through configs on the bloom module.
```

Comment:
```suggestion
Since bloom filters have a default expansion of 2, this means all default bloom filter created by default will be scaling. Additionally, the other default properties of a bloom filter creation can be seen in the table above and BF.INFO command response below. These default properties can be configured through configs on the bloom module.
```